examples/siamese/mnist_siamese_local_solver.prototxt

```diff
@@ -0,0 +1,25 @@
+# The train/test net protocol buffer definition
```

Comment:
I think the local LeNet example suffices, if any is needed at all, and there isn't a need to duplicate it for a siamese net too.